tools/docker/images/builder/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:18.04
+
+ENV https_proxy=http://proxy-iil.intel.com:911
+ENV http_proxy=http://proxy-iil.intel.com:911
+ENV no_proxy=localhost,127.0.0.0/8,192.168.0.0/16,.intel.com,.lantiq.com
+ENV ftp_proxy=http://proxy-iil.intel.com:911
+
+# Update Software repository
+RUN apt-get update && apt-get install -y \
+	curl \
+	gcc \
+	cmake \
+	ninja-build \
+	binutils \
+	git \
+	autoconf \
+	autogen \
+	libtool \
+	pkg-config \
+    	libreadline-dev \
+	libncurses-dev \
+	libssl-dev \
+	libjson-c-dev \
+	libnl-genl-3-dev \
+	libzmq3-dev \
+    	python \
+	python-yaml \
+	repo \
+	vim \
+    && rm -rf /var/lib/apt/lists/*
+
+
+# Set up git and google repo
+WORKDIR /root/prplMesh
+RUN git config --global user.email prplMesh@prpl.com && \
+	git config --global user.name "prplMesh" && \
+	repo init -u https://github.com/prplfoundation/prplMesh-manifest.git --no-clone-bundle && \
+	cp /root/prplMesh/.repo/repo/repo /usr/bin/repo && \
+	repo sync && repo forall -p -c 'git checkout $REPO_RREV'
+
+COPY build.sh /root/prplMesh
+
+ENTRYPOINT ["/root/prplMesh/build.sh"]
+CMD ["build"]
+EXPOSE 80 443
+
+CMD 
```

Comment:
Why do you add a second CMD? It just overrides the default "build" argument, so the first one becomes useless.

After testing, I understood that the `build.sh` doesn't take arguments, so both CMDs are actually useless.